sycl/test-e2e/no-xfail-without-tracker.cpp

```diff
@@ -47,4 +51,168 @@
 // tests to match the required format and in that case you should just update
 // (i.e. reduce) the number below.
 //
-// NUMBER-OF-XFAIL-WITHOUT-TRACKER: 170
\ No newline at end of file
+// NUMBER-OF-XFAIL-WITHOUT-TRACKER: 162
+//
+// List of improperly "xfailed" tests.
+// Remove the CHECK-TEST-NAME once the test has been propely "xfailed".
+//
+// CHECK-TEST-NAME: Basic/stream/auto_flush.cpp
+// CHECK-TEST-NAME: Basic/accessor/accessor.cpp
+// CHECK-TEST-NAME: Basic/max_linear_work_group_size_props.cpp
```

Comment:
We should use `-NEXT` to be strict. Or otherwise someone may introduce a new `XFAIL`, but this list check won't fail

Reply:
We still check the number of improperly XFAIL-ed tests, do you think it's not enough? For some reason the sequence of names after grep is different when running in CI vs. what I see locally. So I'd prefer to have "CHECK-DAG" for now, may be later we'll find a better solution.

Reply:
Well, considering that we do not allow the list to grow, the fact that we don't provide an easy mechanism to find that new "untracked" XFAIL doesn't seem to be important. And it is likely one of the files which are modified by a developer anyway, so it should be easy to track it down.

In that context, `-DAG` should be fine, I think

Reply:
Okay, looks like it works with check-next now: [7e3dfdb](https://github.com/intel/llvm/pull/15801/commits/7e3dfdb68d40fdc6621ed452c5a2685b553d02fe)